src/LinSys/LinSysMerger.h

```diff
@@ -1,7 +1,7 @@
 // *****************************************************************************
 /*!
   \file      src/LinSys/LinSysMerger.h
-  \author    J. Bakosi
+  \author    Los Alamos National Laboratory
```

Comment:
I like the _original_ author (not a list of authors) showing up directly in the source file as well as in the doxygen documentation page. That does not even require a `git blame`. (Some folks do not even know about that git command.)

The rationale for the _original_ author (and not a list) is that that person is most likely to be the one familiar with the design of (and the need for) a given file. `git blame` shows who edited every single line in the file -- a very different, and in some cases way too much, information.